simplq/src/components/common/CreateJoinForm/CreateJoinForm.jsx

```diff
@@ -1,71 +1,88 @@
-import React, { useState } from 'react';
-import { useHistory } from 'react-router';
-import { handleEnterPress } from 'utils/eventHandling';
-import { isQueueNameValid } from 'utils/textOperations';
-import { useCreateQueue } from 'store/asyncActions';
-import { useDispatch } from 'react-redux';
-import LoadingStatus from 'components/common/Loading';
-import styles from './CreateJoinForm.module.scss';
-import InputField from '../InputField';
-import StandardButton from '../Button';
+import React, { useState } from 'react'
+import { useHistory } from 'react-router'
+import { handleEnterPress } from 'utils/eventHandling'
+import { isQueueNameValid } from 'utils/textOperations'
+import { useCreateQueue } from 'store/asyncActions'
+import { useDispatch } from 'react-redux'
+import LoadingStatus from 'components/common/Loading'
+import styles from './CreateJoinForm.module.scss'
+import InputField from '../InputField'
+import StandardButton from '../Button'
+import QRscanner from 'components/common/QR_Scanner/QR_Scanner'
 
```

Comment:
Some files have semicolons on lines, while others don't. Ideally, this shouldn't be the case. I don't really have a preference either way, but since there are more files with semicolons, let's stick to that format. So, please add semicolons in the files where you've made changes.